spec/index.html

```diff
@@ -1021,11 +1021,13 @@ <h2>RDF Datasets</h2>
       can avoid interoperability issues by not ascribing importance to
       the presence or absence of empty named graphs.</p>
 
-    <p>SPARQL 1.2 [[SPARQL12-QUERY]] uses 
+    <p>Version 1.2 of the SPARQL Query Language [[SPARQL12-QUERY]] uses 
       <a data-cite="SPARQL12-QUERY#sparqlDataset">the same concept of an RDF Dataset</a>
+      as RDF versions 1.1 and 1.2,
       in which the graph names of named graphs may be IRIs or blank nodes.
       In contrast, version 1.1 of the SPARQL Query Language [[SPARQL11-QUERY]]
-      only allows IRIs as graph names.</p>
+      and RDF version 1.0
```

Comment:
```suggestion
```

Reply:
done in [4e40fdd](https://github.com/w3c/rdf-concepts/pull/151/commits/4e40fdd369ca046550bdc95535d4fcbad60828b8)